source/general/pkteam.rst

```diff
@@ -0,0 +1,127 @@
+.. _pkteam:
+
+=================
+ProjectKorra Team
+=================
+
+The ProjectKorra team is comprised of volunteers who work on various aspects of the plugin! Many of these volunteers are developers, but there are several other positions!
+
+Meet the Team
+=============
++-----------------+-----------------+---------------+
+| Member          | Position        | MC Avatar     |
++-----------------+-----------------+---------------+
+| Mist            | Admin & Founder | |mist|        |
++-----------------+-----------------+---------------+
+
+Developers
+----------
+
+Developers are the people who maintain and work on the various
+development projects that require programming knowledge
+
++-----------------+-----------------+---------------+
+| OmniCypher      | Project Lead    | |omnicypher|  |
++-----------------+-----------------+---------------+
+| Sobki           | Lead Developer  | |sobki|       |
++-----------------+-----------------+---------------+
+| Coolade         | Developer       | |coolade|     |
++-----------------+-----------------+---------------+
+| Simplicitee     | Developer       | |simplicitee| |
++-----------------+-----------------+---------------+
+| Jacklin213      | Developer       | |jacklin|     |
++-----------------+-----------------+---------------+
+| AlexTheCoder    | Developer       | |alex|        |
++-----------------+-----------------+---------------+
+| StrangeOne101   | Developer       | |strange|     |
++-----------------+-----------------+---------------+
+| PhanaticD       | Developer       | |phanatic|    |
++-----------------+-----------------+---------------+
+| Varhagna        | Developer       | |varhagna|    |
++-----------------+-----------------+---------------+
+| Finn\_Bueno_    | Developer       | |finnbueno|   |
++-----------------+-----------------+---------------+
+| plasmarob       | Developer       | |plasmarob|   |
++-----------------+-----------------+---------------+
+| FavouriteDragon | Forge Developer | |favourite|   |
++-----------------+-----------------+---------------+
+| Mahtaran        | Forge Developer | |mahtaran|    |
++-----------------+-----------------+---------------+
+| Mnesikos        | Forge Developer | |mnesikos|    |
++-----------------+-----------------+---------------+
+| TheWaterChief   | Bug Tester      | |waterchief|  |
++-----------------+-----------------+---------------+
+
+Moderators
+----------
+
+Moderators work on the forums and discord to keep them
+organized, civil, and spam-free.
+
++-----------------+-----------------+---------------+
+| Pickle9775      | Head Moderator  | |pickle|      |
++-----------------+-----------------+---------------+
+| floory565       | Moderator       | |floory|      |
++-----------------+-----------------+---------------+
+
```

Comment:
StrangeOne101 and I are moderators as well

Reply:
I only put people in their main team

Reply:
okay